### ISSUE TITLE
Right root, right projects and the wrong tsconfig file is loaded instead in V4

### ISSUE THREAD

**Issue by MohamedLamineAllal** [aleclarson/vite-tsconfig-paths#83]
# Description

tsconfig resolution is not working correctly in `v4.0.0`

The same vite config file works in `v3.6` and not in `v4`. Here the demonstration bellow:

## v4 output

We can see in the log bellow that `root` and `projects` are passed correctly. But the loaded file is the wrong one instead (`tsconfing.node.json`).

```sh
playground-1 DEBUG=vite-tsconfig-paths yarn vite
yarn run v1.22.19
$ /Users/mohamedlamineallal/repos/playground/Javascript/react/react-18/typescript/native-base/vite/playground-1/node_modules/.bin/vite
  vite-tsconfig-paths options.root   == /Users/mohamedlamineallal/repos/playground/Javascript/react/react-18/typescript/native-base/vite/playground-1 +0ms
  vite-tsconfig-paths project root   == /Users/mohamedlamineallal/repos/playground/Javascript/react/react-18/typescript/native-base/vite/playground-1 +0ms
  vite-tsconfig-paths workspace root == undefined +0ms
  vite-tsconfig-paths projects: [
  '/Users/mohamedlamineallal/repos/playground/Javascript/react/react-18/typescript/native-base/vite/playground-1/tsconfig.json'
] +1ms
  vite-tsconfig-paths config loaded: {
  configPath: '/Users/mohamedlamineallal/repos/playground/Javascript/react/react-18/typescript/native-base/vite/playground-1/tsconfig.node.json',
  config: {
    compilerOptions: {
      composite: true,
      module: 'ESNext',
      moduleResolution: 'Node',
      allowSyntheticDefaultImports: true
    },
    include: [ 'vite.config.ts' ]
  }
} +3ms
  vite-tsconfig-paths [!] missing baseUrl and paths: "/Users/mohamedlamineallal/repos/playground/Javascript/react/react-18/typescript/native-base/vite/playground-1/tsconfig.node.json" +0ms

  VITE v3.2.5  ready in 270 ms
```

Dependencies:

```json
"vite": "^3.2.3",
"vite-tsconfig-paths": "~4.0.0"
```

**Comment by aleclarson** [comment]
Should be fixed in v4.0.1

**Comment by MohamedLamineAllal** [comment]
Install with the following:

```sh
npm install vite-tsconfig-paths@latest
```




  [1]: https://www.npmjs.com/package/vite-tsconfig-paths
